src/app/views/tags/index.html.erb

```diff
@@ -0,0 +1 @@
+<h2>Under Construction</h3>
```

Comment:
👀 so that will be the tags list ordered alphabetically right?

Reply:
Whatever you sees fit, cap!